src/content/lodash/function/after/vanilla.js

```diff
@@ -0,0 +1,14 @@
+const after = (times, fn) => {
+  let counter = 0
+
+  return (...args) => {
+    counter++
+
+    if (counter >= times) {
+      return fn(...args)
+    }
```

Comment:
technically, this can be simplified to this (not sure it's more readable though)

```suggestion
    if (++counter >= times) {
      return fn(...args)
    }
```

Reply:
good shout but I'd keep readability over terseness in this case